roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- include: ./checks/check_system.yml
+
+- include: ./checks/check_mandatory_vars.yml
+
+- include: ./misc/system_tuning.yml
+  when: osd_group_name in group_names
+
+- include: ./installs/install_on_debian.yml
+  when: ansible_os_family == 'Debian'
+
+- name: check for a ceph socket
+  stat: path=/var/run/ceph/*.asok
+  changed_when: false
+  failed_when: false
+  register: socket
+
+- name: create a local fetch directory if it doesn't exist
+  local_action: file path=fetch state=directory 
+  changed_when: false
+  sudo: false
+
+- name: generate cluster uuid
+  local_action: shell uuidgen | tee fetch/ceph_cluster_uuid.conf
+                creates=fetch/ceph_cluster_uuid.conf
+  register: cluster_uuid
+  sudo: false
+  when: fsid != '4a158d27-f750-41d5-9e7f-26ce4c9d2d45'
+
+- name: read cluster uuid if it already exists
+  local_action: command cat fetch/ceph_cluster_uuid.conf
+                removes=fetch/ceph_cluster_uuid.conf
+  changed_when: false
+  register: cluster_uuid
+  sudo: false
+  when: fsid != '4a158d27-f750-41d5-9e7f-26ce4c9d2d45'
```

Comment:
Looks like this is trying to determine what the cluster ID is, but that could be discoverable on the host itself, rather than checking a local file's existence. We really don't like the idea of writing to a local path for this, as multiple people could be running the playbook and they wouldn't have the same local untracked files.